docs/userGuide/plugins/dataTable.md

```diff
@@ -0,0 +1,76 @@
+### Plugin: DataTable
+
+The DataTable plugin enhances the functionality of tables in your MarkBind site by integrating the DataTables library. It allows you to add searching and sorting capabilities to your tables with minimal configuration. The necessary CSS and JavaScript files are already included in the project, so no additional CDN or plugin context configuration is required.
+
+To enable this plugin, simply add `datatable` to your site's plugins:
+
+```js {heading="site.json"}
+{
+  ...
+  "plugins": [
+    "datatable"
+  ]
+}
+```
+
+To create a table with DataTable features, use the following syntax:
+
+```html
+<m-table sortable>
```

Comment:
Hi @Tim-Siu this seems a bit confusing for the user that the plugin is dataTable but the syntax is m-table. Could we standardise this?

Reply:
Could you also use both sortable and searchable here since it should be a syntax example?

Reply:
What about changing it to <d-table>?

Reply:
> Could you also use both sortable and searchable here since it should be a syntax example?

I have made changes accordingly. Thanks for your suggestions

Reply:
> What about changing it to `<d-table>`?

This sounds good! Thank you